sstables/sstable_directory.hh

```diff
@@ -53,7 +53,7 @@ class sstable_directory {
 public:
     // favor chunked vectors when dealing with file lists: they can grow to hundreds of thousands
     // of elements.
-    using sstable_info_vector = utils::chunked_vector<sstables::foreign_sstable_open_info>;
+    using sstable_open_info_vector = utils::chunked_vector<sstables::foreign_sstable_open_info>;
```

Comment:
might need to update `replica/distributed_loader.cc` also.

Reply:
indeed. one of the reasons for compilation failure. thanks.